apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -187,6 +187,11 @@ class ManageStudentsTable extends Component {
     return userType === 'teacher';
   }
 
+  // Showing secret picture is disabled if the "student" in the section is a teacher
+  isSecretLoginDisabled(userType) {
```

Comment:
This function is the same as the function above on line 186, just with a different name. I'm going to combine these into a single function named `isTeacher`. Will push the change up shortly.